main.py

```diff
@@ -1399,6 +1444,7 @@ def update_file_diagnostics(window: sublime.Window, file_path: str, source: str,
 
 
 def update_diagnostics_phantoms(view: sublime.View, diagnostics: 'List[Diagnostic]'):
+    return
```

Comment:
Why do we return here from this function immediately?

Reply:
Ah... I will delete that. I was having issues with the Phantoms, so I forced LSP's existing phantoms to no longer show up, just in case that was causing the issue, but I forgot to remove that return statement. I don't think this was the cause of the issue, so I'll re-test stuff with that return statement removed.

Reply:
So, I tested, and this `update_diagnostics_phantoms` function is actually causing issues. My phantoms will show up for a short time, but as soon as this function gets triggered, it clears all Phantoms from the document, even though (or especially since) I have `"show_diagnostics_phantoms": false` in my config file.

Adding some more config options to control the new features I introduced for the Rust side is on my todo list.

Reply:
But, the PhantomSet's have different ID strings, so I'm not sure why they interfere with each other.